### PR TITLE
Generate ComboBox::set_active by hand

### LINKED ISSUE #749: Combobox::set_active should take an option
The docs say that [set_active](https://gtk-rs.org/docs/gtk/trait.ComboBoxExt.html#tymethod.set_active) will deactivate the current one if the given value is `-1`. `Option` seems better in rust case.

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Fixes #749 

cc @EPashkin @sdroege

### CODE REVIEW

File: src/combo_box.rs [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// Copyright 2018, The Gtk-rs Project Developers.
+// See the COPYRIGHT file at the top-level directory of this distribution.
+// Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
+
+use ComboBox;
+use ffi;
+use glib::object::IsA;
+
+pub trait ComboBoxExtManual {
+    fn set_active<'a, P: Into<Option<u32>>>(&self, index_: P);
```
Comment (EPashkin) [inline src/combo_box.rs]: `'a` is really needed ?

**Comment by EPashkin** [comment]
`ComboBoxExtManual` need be in prelude, and old function ignored

File: src/combo_box.rs (hunk 1 above)
Comment (sdroege) [reply src/combo_box.rs]: No, and I'm surprised it's not a compiler warning :)

**Comment by sdroege** [comment]
looks good to me apart from what @EPashkin said

**Comment by GuillaumeGomez** [comment]
I completely forgot to commit all files, sorry...

**Comment by EPashkin** [comment]
Why don't ignore property getter and setter?
```
    [[object.property]]
    name = "active"
    #manual Option
    generate = ["notify"]
```

File: src/auto/combo_box.rs [hunk 2]
```diff
@@ -447,6 +435,20 @@ impl<O: IsA<ComboBox> + IsA<glib::object::Object> + glib::object::ObjectExt> Com
         }
     }
 
+    fn get_property_active(&self) -> i32 {
```
Comment (sdroege) [inline src/auto/combo_box.rs]: The property functions should disappear (but notify should stay), otherwise looks good to me

**Comment by EPashkin** [comment]
@GuillaumeGomez thanks, 👍

**Pull request merged**